scripts/update_images.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# -*- coding: utf-8 -*-
```

Comment:
- `coding` is not necessary, a reminder from Python I guess?
- There are quite a few warnings if you run `shellcheck`, but we can fix this later.

Reply:
Yes, it was a reminder. Regarding spellcheck... I have the linter activated in the editor, but since the last upgrade, I have no spellcheck anymore... I've installed it again, thanks for the heads-up!